lib/order.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/menu'
+class Order
+  attr_reader :order_price, :valid_item
+  def initialize
+    @menu = Menu.new
+   # @basket = Basket.new
+    @basket = Basket.new
+    @order_price = 0
+  end
+
+  def select(*items)
```

Comment:
Think about edge cases here. E.g, if the user input a valid dish, but used Capitals. Possibly use .downcase on the parameter to account for this?